composer.json

```diff
@@ -16,7 +16,7 @@
         "padraic/humbug_get_contents": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "^4.0|^5.7|^6.0"
```

Comment:
@theofidry Oh, it's dead after next version here. I'm just loathe to leave downstream without a SHA256 option, even if they are still on EOL PHP versions. Once we're done there, we move directly to whatever we determine as min PHP for Humbug 2.0.